procrastinate/cli.py

```diff
@@ -77,13 +69,11 @@ def print_version(ctx, __, value):
     count=True,
     help="Use multiple times to increase verbosity",
 )
-@click.option(
-    "-V",
-    "--version",
-    is_flag=True,
-    callback=print_version,
-    expose_value=False,
-    is_eager=True,
+@click.version_option(
+    procrastinate.__version__,
+    '-V',
+    '--version',
+    prog_name=PROGRAM_NAME
```

Comment:
Curiosity: why is `prog_name` required?

Reply:
For the test_version. Without it, it thinks cli is the program name so the test was broken. 2 choiches, define prog_name here or change the test to assert on cli instead of procrastinate prog_name

Reply:
nah, I like it that way.